pkgs/by-name/wo/wordlists/package.nix

```diff
@@ -1,45 +1,61 @@
 { lib
-, callPackage
+, symlinkJoin
 , nmap
 , rockyou
-, runtimeShell
 , seclists
-, symlinkJoin
-, tree
 , wfuzz
 , lists ? [
     nmap
     rockyou
     seclists
     wfuzz
   ]
+, writeShellScriptBin
+, tree
 }:
+let
+  version = "unstable-2023-11-14";
```

Comment:
version bump should be indicated in the commit message

Reply:
The versioning is a bit weird, since this is a sort of metapackage. Is the current commit message okay?

Reply:
> The versioning is a bit weird, since this is a sort of metapackage.

Yes, the only other precedent like this I know is [nixos-firewall-tool](https://search.nixos.org/packages?channel=unstable&show=nixos-firewall-tool&from=0&size=250&sort=relevance&type=packages&query=nixos-firewall-tool) and it just doesn't have a version entry so we could probably just drop the version attribute here as well since symlinkJoin doesn't require it.

Also I wouldn't really count this as version bump since it's not a breaking change, it's also not a new feature, and we also don't fix a bug, it's just changing some background wiring.

Reply:
I have removed the version tag, updated the commit message to reflect that, and rebased against master